lib/core/auth.js

```diff
@@ -188,44 +193,63 @@ export default class Auth {
   // Token helpers
   // ---------------------------------------------------------------
 
-  getToken (strategy) {
-    const _key = this.options.token.prefix + strategy
+  getToken () {
+    return this._token.get()
+  }
 
-    return this.$storage.getUniversal(_key)
+  setToken (token) {
+    return this._token.set(token)
   }
 
-  setToken (strategy, token) {
-    const _key = this.options.token.prefix + strategy
+  syncToken () {
+    return this._token.sync()
+  }
 
-    return this.$storage.setUniversal(_key, token)
+  resetToken () {
+    return this._token.reset()
   }
 
-  syncToken (strategy) {
-    const _key = this.options.token.prefix + strategy
+  getTokenStatus () {
+    return this._token.getStatus()
+  }
 
-    return this.$storage.syncUniversal(_key)
+  refreshTokenAt () {
+    return this._token.refreshAt()
   }
 
   // ---------------------------------------------------------------
   // Refresh token helpers
   // ---------------------------------------------------------------
 
-  getRefreshToken (strategy) {
-    const _key = this.options.refresh_token.prefix + strategy
+  getRefreshToken () {
+    return this._refreshToken.get()
```

Comment:
Same for refreshToken :)

Reply:
Ok!

Reply:
Refactored in commits https://github.com/nuxt-community/auth-module/commit/76e322166097145c73f17ec91190dae2416cbda9 and https://github.com/nuxt-community/auth-module/commit/558dfa7c5bb6f65e02b2449028ef1c656c5bfda2